Pandemic/Board.cpp

```diff
@@ -206,6 +210,97 @@ void Board::setPlayerCardsFromLoad() {
 	cardManager->moveCardToDeck();
 }
 
+void Board::epidemicCardAction() {
+	incrementInfectionRate();
+	bottomInfectionCard = cardManager->getInfectionCardDeck()->at(cardManager->getInfectionCardDeck()->size() - 1);
```

Comment:
This should be random since we never "shuffle" the deck. we just randomly pick a card. Doing what you're currently doing doesn't take into account the "shuffling" of cards that occurs.